documentation/docs/03-template-syntax/01-basic-markup.md

```diff
@@ -154,6 +177,26 @@ A JavaScript expression can be included as text by surrounding it with curly bra
 {expression}
 ```
 
+When using {expression} inside markup, Svelte automatically converts the value to a string before rendering it. The conversion follows JavaScript's standard behavior:
+
+- Primitive values (number, boolean, string) are directly converted to strings.
+- Objects call their .toString() method (if not overridden, it defaults to [object Object]).
+- undefined and null are treated as empty strings ("").
+
+```svelte
+<script>
+	let num = 1;
+	let bool = false;
+	let obj = { key: "value" };
+	let empty = undefined;
+</script>
+
+<p>{num}</p>   <!-- Renders as: <p>1</p> -->
+<p>{bool}</p>  <!-- Renders as: <p>false</p> -->
+<p>{obj}</p>   <!-- Renders as: <p>[object Object]</p> -->
+<p>{empty}</p> <!-- Renders as: <p></p> (empty string) -->
```

Comment:
Add an example with an empty string, null, and an object with the toString method.
Perhaps it's better to shorten the example by removing the script tag.